src/store/apis/hotels/searchHotels.endpoint.js

```diff
@@ -0,0 +1,16 @@
+import { apiEndpoint } from '../../../constants/path';
+
+import { homesApi } from './getPopularHotels.endpoint';
+
+export const searchApi = homesApi.injectEndpoints({
```

Comment:
No need to export this one
```suggestion
const searchApi = homesApi.injectEndpoints({
```